RealTimeFaceMaskDetector/TCPClientUI/TCPClientUI.cpp

```diff
@@ -1,5 +1,6 @@
 #include "TCPClientUI.h"
 #include "TCPClient.h"
+#include "../EncryptDecryptECBMode/EncryptDecryptAES_ECBMode.h"
 
```

Comment:
Here add a description, or at least definition location too

Reply:
Added description for global variables